src/waitForTrigger.m

```diff
@@ -45,7 +45,7 @@ function waitForTrigger(varargin)
 
             keyCode = []; %#ok<NASGU>
 
-            [~, ~, keyCode] = KbCheck(deviceNumber);
+            [~, lastTriggerTimeStamp, keyCode] = KbCheck(deviceNumber);
```

Comment:
yes this is exactly what I'm / was doing! thanks! I didn't update wirh the previous PR so I was using:
`[sec, keyCode] = KbPressWait(deviceNumber);`

Reply:
okidoki so I will merge this now. :-)